ui-components/src/commonMain/kotlin/org/jetbrains/kotlinconf/ui/components/ServiceEvents.kt

```diff
@@ -0,0 +1,145 @@
+package org.jetbrains.kotlinconf.ui.components
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import org.jetbrains.compose.ui.tooling.preview.Preview
+import org.jetbrains.kotlinconf.ui.components.NowLabel
+import org.jetbrains.kotlinconf.ui.components.StyledText
+import org.jetbrains.kotlinconf.ui.theme.KotlinConfTheme
+import org.jetbrains.kotlinconf.ui.theme.PreviewHelper
+
+data class ServiceEventData(
+    val name: String,
+    val now: Boolean,
+    val note: String? = null,
+    val time: String? = null,
+)
+
+@Composable
+private fun ServiceEventItem(
+    event: ServiceEventData,
```

Comment:
The packed properties look inconsistent with other components. Could you tell me what is the reason behind this?

Reply:
fair, let's keep it